src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Excavator extends Machine{
+    @Override
+    public void doWork() {
+        System.out.println("Excavator started working");

```

Comment:
The message in `System.out.println` should be more informative and indicate that it is an `Excavator` that has started working, as per the checklist's requirement to use informative messages.